src/Serilog.Extensions.Hosting/Serilog.Extensions.Hosting.csproj

```diff
@@ -38,9 +38,9 @@
 
   <ItemGroup>
     <!-- The versions of all references in this group must match the major and minor components of the package version prefix. -->
-    <PackageReference Include="Serilog.Extensions.Logging" Version="8.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.0" />
+    <PackageReference Include="Serilog.Extensions.Logging" Version="9.0.0" />
```

Comment:
This version currently does not exist - there's a PR for it in [serilog-extensions-logging](https://github.com/serilog/serilog-extensions-logging/pull/263).